src/tm-threads.c

```diff
@@ -366,6 +366,93 @@ static void *TmThreadsSlotPktAcqLoop(void *td)
     return NULL;
 }
 
+static void *TmThreadsLib(void *td)
```

Comment:
why do we need this? Missing a comment explaining its purpose. Also can it be (partly) deduplicated with other funcs in the file?

Reply:
This is very similar to "TmThreadsSlotPktAcqLoop" but it does not invoke the slots "PktAcqLoop" method.